distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -179,6 +182,31 @@ Note that for all RESTful CRUD commands only one metacard ID is supported in t
 Any web browser can be used to perform a REST read.
 Various other tools and libraries can be used to perform the other HTTP operations on the REST endpoint (e.g., soapUI, cURL, etc.)
 
+The REST endpoint can be used to upload resources as attachments. The `create` and `update` methods both support the multipart mime format. If only a single attachment exists, it will be interpreted as a resource to be parsed, which will result in a metacard and resource being stored in the system.
+
+If multiple attachments exist, then the REST endpoint will assume that 1 attachment is the actual resource (attachment should be named `parse.resource`) and the other attachments are overrides of metacard attributes (attachment names should follow metacard attribute names). In the case of the metadata attribute, it is possible to also have the system transform that metadata and use the results of that to override the metacard that would be generated from the resource (attachment should be named `parse.metadata`).
+
+For example:
+[source,http]
+----
+POST /services/catalog?transform=xml HTTP/1.1
+Host: localhost:8993
+Content-Type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW
+Cache-Control: no-cache
+
+------WebKitFormBoundary7MA4YWxkTrZu0gW
+Content-Disposition: form-data; name="parse.resource"; filename=""
+Content-Type:
+
+
+------WebKitFormBoundary7MA4YWxkTrZu0gW
+Content-Disposition: form-data; name="parse.metadata"; filename=""
+Content-Type:
+
+
+------WebKitFormBoundary7MA4YWxkTrZu0gW--
+----
+
```

Comment:
I'm copying these changes into the draft-documentation version too, so they are not lost when we transition.